test/test_jld2_output_writer.jl

```diff
@@ -186,6 +186,140 @@ function test_jld2_time_averaging_of_horizontal_averages(model)
     return nothing
 end
 
+function test_jld2_time_averaging(arch)
+    # Test for both "nice" floating point number and one that is more susceptible
+    # to rounding errors
+    for Δt in [1/64, 0.01]
```

Comment:
```suggestion
    for Δt in (1/64, 0.01)
```

just asthetics, but a tuple is better than an array